buildtools/webpack.commons.js

```diff
@@ -119,7 +119,9 @@ module.exports = function (config) {
   }
 
   const ngeoRule = {
-    test: /\/ngeo\/(?!node_modules\/).*\.js$/,
+    test: (item) => {
```

Comment:
If you revert this change and the next one it works again (at least the apps.)
I don't know exactly why there is this issue but that looked possibly errored.
I've a meeting all this morning, I can't investigate more right now

Reply:
So basically, the path (item) start with /home/.... locally. Never with `ngeo/` or `nodes_modules/`, etc.
I'll try to know what we want to collect, comment it and fix the regexp